src/reference/v8.json

```diff
@@ -17,7 +17,17 @@
     },
     "metadata": {
       "type": "*",
-      "doc": "Arbitrary properties useful to track with the stylesheet, but do not influence rendering. Properties should be prefixed to avoid collisions, like 'maplibre:'."
+      "doc": "Arbitrary properties useful to track with the stylesheet, but do not influence rendering. Properties should be prefixed to avoid collisions, like 'maplibre:'.",
+      "example": {
+        "maplibre:autocomposite": true,
+        "fresco:comment": "Style generated by https://fresco.go-spatial.org",
+        "maputnik:version": "3.14.159265",
+        "maplibre:sdk-support": {
```

Comment:
This isn't a great example as reading this when looking at the v8.json file can be confusing.
I would also avoid using external url and names here to avoid infringing copy right laws...
Keep it simple...

Reply:
Which line are you referring to?  I'm happy to get this right.

Reply:
`example:sdk-support` I would consider replacing it with `"example:object": { a: "1", b: 2, c: false }`

Reply:
Excellent!  Here is the change:  https://github.com/maplibre/maplibre-gl-style-spec/pull/34/commits/23f539844c5c91be731d2b3419963953a2bdb58b